### PR TITLE
adding blog post: Why I used GatsbyJS

### PR DESCRIPTION
**Pull request by ronthomas345**
Why I used GatsbyJS

### CODE REVIEW

**Comment by ronthomas345** [comment]
@jlengstorf ,please review this PR.

File: docs/docs/awesome-gatsby.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ See the [library of official and community plugins](/plugins/)
 - [Zero to Deploy: A Practical Guide to Static Sites with Gatsby.js](https://scotch.io/tutorials/zero-to-deploy-a-practical-guide-to-static-sites-with-gatsbyjs)
 - [Build blazing fast web applications with Gatsby 101](https://gatsbyguides.com/)
 - [Build a Coffee Shop Website w/ Gatsby, Contentful, SnipCard & Formspree](https://www.youtube.com/watch?v=oAVhEPey_qA)
+-[Why I used GatsbyJS](https://medium.com/@jameshunt.co/why-i-used-gatsbyjs-70bbb0f5c4a7)
```
Comment (muescha) [inline docs/docs/awesome-gatsby.md]:
```suggestion
- [Why I used GatsbyJS](https://medium.com/@jameshunt.co/why-i-used-gatsbyjs-70bbb0f5c4a7)
```

**Comment by ronthomas345** [comment]
@jlengstorf ,please merge the PR.

**Comment by jlengstorf** [comment]
We need approval from @gatsbyjs/docs, then it's good to go in.

**Comment by marcysutton** [comment]
Just waiting on the lint test since I'm OCD about it passing, should be good to go shortly. 😄

**Comment by DSchau** [comment]
@marcysutton I'm glad you did--since I think it was technically failing! If the linter passes -> 🚀

**Pull request merged**